internal/cmd/config/config.go

```diff
@@ -115,6 +115,13 @@ type Controller struct {
 	// denoted by time.Duration
 	AuthTokenTimeToStale         interface{} `hcl:"auth_token_time_to_stale"`
 	AuthTokenTimeToStaleDuration time.Duration
+
+	// StatusGracePeriod represents the period of time (as a duration) that the
+	// controller will wait before marking connections from a disconnected worker
+	// as invalid.
+	//
+	// TODO: This field is currently internal.
+	StatusGracePeriodDuration time.Duration `hcl:"-"`
```

Comment:
I don't think we'll ever want this value exposed via config, so rather than put it in here, can you simple set it as a field on the `base.Server` and then it can be toggled in tests from there?

Reply:
I think this can now be removed here and in the Worker block right?